src/main/java/com/beust/jcommander/Parameterized.java

```diff
@@ -121,6 +121,9 @@ public static List<Parameterized> parseArg(Object arg) {
       // check methods
       for (Method m : cls.getDeclaredMethods()) {
         m.setAccessible(true);
+        if (m.isBridge() || m.isSynthetic()) {
```

Comment:
I think it is a good idea to exclude these methods always. Since these methods don't actually appear in the source code, the error that there is a duplicate method is quite surprising to most developers (I was quite confused until I realized that it was a bridge method created by the compiler). I can't actually imagine a scenario where you would want to take a synthetic or bridge method into consideration in this code.

Reply:
While I tend to see it your way, I am just not convinced that excluding *non-duplicates* would be correct (as we only have a problem with the fact that it is a duplicate). In fact, I wonder why bridged and synthetic methods are reported as *declared* methods -- nobody *declared* them. 🤔

Reply:
Updated the logic here to include bridge and synthetic methods if they are not a duplicate of an existing "normal" method. Thanks.